benchmark/opperf/rules/default_params.py

```diff
@@ -14,10 +14,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import sys
+
+# We will use all operators inside NDArray Module
+# If you want to run benchmark for all operators in different namespace,
+# for example mxnet.numpy.op, update here. All operators for benchmarks
+# will be picked up from this module
+MX_OP_MODULE = sys.modules["mxnet.ndarray.op"]
```

Comment:
Can we make this configurable at runtime? It will be easier for CI team to automate the comparison between numpy operators and ndarray operators, right?

Reply:
Agree. Will take it as TODO?